parse.go

```diff
@@ -23,6 +26,8 @@ func FromString(r io.Reader, chunker string) (Splitter, error) {
 		size, err := strconv.Atoi(sizeStr)
 		if err != nil {
 			return nil, err
+		} else if size <= 0 {
+			return nil, ErrSize
```

Comment:
This is fine.  I think it would be better though to have `NewSizeSplitter` verify its arguments are sane and return an error.  But since NewRabinMinMax does not return an error it is not that big of a deal.